documentation/book/appendix_crds.adoc

```diff
@@ -419,32 +419,98 @@ Used in: <<type-KafkaConnect,`KafkaConnect`>>
 
 |https://v1-9.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.9/#tolerations-v1-core[Toleration] array
 |logging           1.2+<.<|Logging configuration for Kafka Connect The type depends on the value of the `logging.type` property within the given object, which must be one of [inline, external]
-|<<type-InlineLogging,`InlineLogging`>>, <<type-ExternalLogging,`ExternalLogging`>>
+|xref:type-InlineLogging-{context}[`InlineLogging`], xref:type-ExternalLogging-{context}[`ExternalLogging`]
 |metrics           1.2+<.<|The Prometheus JMX Exporter configuration. See https://github.com/prometheus/jmx_exporter for details of the structure of this configuration.
 |map
+|authentication    1.2+<.<|Authentication configuration The type depends on the value of the `authentication.type` property within the given object, which must be one of [tls]
+|xref:type-KafkaConnectAuthenticationTls-{context}[`KafkaConnectAuthenticationTls`]
 |bootstrapServers  1.2+<.<|Bootstrap servers to connect to. This should be given as a comma separated list of _<hostname>_:‍_<port>_ pairs.
 |string
 |config            1.2+<.<|The Kafka Connect configuration. Properties with the following prefixes cannot be set: ssl., sasl., security., listeners, plugin.path, rest., bootstrap.servers
 |map
 |resources         1.2+<.<|Resource constraints (limits and requests).
-|<<type-Resources,`Resources`>>
+|xref:type-Resources-{context}[`Resources`]
+|tls               1.2+<.<|TLS configuration
+|xref:type-KafkaConnectTls-{context}[`KafkaConnectTls`]
 |====
 
-[[type-KafkaConnectS2I]]
-### `KafkaConnectS2I` kind v1alpha1 kafka.strimzi.io
+[id='type-KafkaConnectAuthenticationTls-{context}']
+### `KafkaConnectAuthenticationTls` schema reference
+
+Used in: xref:type-KafkaConnectS2IAssemblySpec-{context}[`KafkaConnectS2IAssemblySpec`], xref:type-KafkaConnectSpec-{context}[`KafkaConnectSpec`]
+
+
+The `type` property is a discriminator that distinguishes the use of the type `KafkaConnectAuthenticationTls` from .
```

Comment:
A small fix to the doc generator needed here. I didn't anticipate people using descriminators in hierarchies with a single concrete class.